styles/feed-event.mcss

```diff
@@ -37,6 +37,10 @@ FeedEvent {
 
   div.meta {
     font-size: 100%
-    padding: 10px
+    padding: 10px 20px
+    opacity: 0.4
+    :hover {
+      opacity: 0.8
```

Comment:
Here's an example of replies to an old post (the old post is no longer showing because it's fallen out of the window):

<img width="739" alt="screen shot 2017-04-24 at 9 19 59 am" src="https://cloud.githubusercontent.com/assets/66834/25317577/5ade8704-28cf-11e7-8bd7-e72f0aac18ef.png">

It's hard to tell in this new design that it is a **reply** and **not a post**.